src/native/external/llvm-libunwind/src/Registers.hpp

```diff
@@ -5268,7 +5268,8 @@ class _LIBUNWIND_HIDDEN Registers_loongarch {
 
   bool validRegister(int num) const;
   uint64_t getRegister(int num) const;
-  void setRegister(int num, uint64_t value);
+  void setRegister(int num, uint64_t value, uint64_t location);
+  uint64_t  getRegisterLocation(int num) const { return 0; }
```

Comment:
It's loongarch-specific section of external code. Have you submitted the change to upstream?

Given the TODO in header file, is this required for features implemented in this PR?

Reply:
These changes are not upstreamable as-is. The `location` parameter is added in the dotnet/runtime downstream changes for all archs.